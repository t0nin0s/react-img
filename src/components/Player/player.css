.name {
  color: #018DE5;
  font-weight: bold;
  font-size: 1.5rem;
}

.country {
  font-size: 1.2rem;
  font-weight:  400;
}
