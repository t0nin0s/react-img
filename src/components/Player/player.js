import React from 'react'

import styles from './player.css'

const Player = (props) => (
  <div className={styles.player}>
    <span className={styles.name}>{props.firstName} {props.lastName}</span>
    <span className={styles.country}> ({ props.country })</span>
  </div>
)

export default Player
