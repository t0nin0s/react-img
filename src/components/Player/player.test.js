import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Player from './player'

const player = {
  id: 2,
  firstName: "Mark",
  lastName: "Steven",
  country: "UK"
}

test('Player component renders as expected', () => {
  const component = shallow(<Player  {...player} />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
