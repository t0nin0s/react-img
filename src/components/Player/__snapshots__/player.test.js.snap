exports[`test Player component renders as expected 1`] = `
<div
  className="player">
  <span
    className="name">
    Mark
     
    Steven
  </span>
  <span
    className="country">
     (
    UK
    )
  </span>
</div>
`;
